src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,3 @@
+.logo__brand-img {
+  display: block;
+}
```

Comment:
Even if you don't style the block, you need to stick to the correct structure of your code
```
.logo {
  &__brand-img {
    display: block;
  }
}
```